waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -74,13 +74,17 @@ type
     # in event of a reorg. we store 5 in the buffer. Maybe need to revisit this,
     # because the average reorg depth is 1 to 2 blocks.
     validRootBuffer*: Deque[MerkleNode]
+    # interval loop to shut down gracefully
+    blockFetchingActive*: bool
 
 const DefaultKeyStorePath* = "rlnKeystore.json"
 const DefaultKeyStorePassword* = "password"
 
+const DefaultBlockPollRate* = 1.seconds
```

Comment:
Perhaps this is too often? Would suggest 6s (half-block time). Thinking here of people that may be using freemium providers. Once per second is 86.400 requests a day (and that just for knowing the current block).

On the other hand, since we consider as valid multiple roots, we could relax this further (thinking of >30sec). But lets leave this by now, as it may open up some edge cases. 6s should be ok by now.

Reply:
addressed in https://github.com/waku-org/nwaku/pull/2364/commits/dbd6cc3617634314295c11f4b19ab064de5eb25a